devops/github-actions/prefect-deploy-only-when-files-change-including-docker-build.yaml

```diff
@@ -0,0 +1,71 @@
+---
+name: Build and publish Prefect Deployments & Docker images
+
+"on":
```

Comment:
I've not see this before

Reply:
yep! `on` in yaml is treated as a `bool`, it doesn't cause failures or anything - but yaml linters will throw warnings